bot/resources/tags/nomodule.md

```diff
@@ -0,0 +1,15 @@
+**ModuleNotFoundError**
+
+If you've installed a package but you're getting a ModuleNotFoundError, it's likely that the environment where your code is running is different from the one where you did the installation.
+
+You can read about Python environments at `!tags environments` and `!tags venv`.
+
+Common causes of this problem include:
+
+• You installed your package using `pip install ...`. It could be that the `pip` command is not pointing the the environment where your code runs. For greater control, you could instead run pip as a module within the python environment you specify:
+```
+python -m pip install <your_package>
+```
+(You may have to specify the full path to the python executable).
```

Comment:
This is slightly ambiguous and may confuse the reader

Reply:
Agreed. It may be best to leave it out unless it can be explained briefly and clearly how "xxx -m pip install ..." means the package is available if using "xxx file.py"/"xxx -m mymodule" to run the code.